dev_guide/cron_jobs.adoc

```diff
@@ -75,9 +75,77 @@ spec:
           - name: pi
             image: perl
             command: ["perl",  "-Mbignum=bpi", "-wle", "print bpi(2000)"]
-          restartPolicy: Never
+          restartPolicy: Never <3>
 ----
 
-1. Schedule for the job. In this example, a job will run every minute.
+1. Schedule for the job. In this example, the job will run every minute.
 2. Job template. This is similar to the xref:../dev_guide/jobs.adoc#creating-a-job[job example].
+3. The restart policy of the pod. This does not apply to the job controller. See xref:../dev_guide/jobs.adoc#creating-a-job-known-issues[Known Issues and Limitations] for details.
 ====
+
+Alternatively, you can create and launch a cron job from a single command -- specifically, `oc run`. The following command creates and launches the same cron job as the `pi` resource:
+
+----
+$ oc run pi --image=perl --schedule='* * * * *' --restart=Never --command -- perl -Mbignum=bpi -wle 'print bpi(2000)' 
+----
+ 
+With `oc run`, the `--schedule` option accepts schedules in link:https://en.wikipedia.org/wiki/Cron[cron format]. 
+
+[NOTE]
+=====
+When creating a cron job,  `oc run` only supports the `Never` or `OnFailure` restart policies (`--restart`).
+=====
+
+[TIP]
+=====
+Delete cron jobs that you no longer need:
+----
+$ oc delete --cronjob <cron_job_name>
+----
+Doing so will prevent them from generating unnecessary artifacts.
+=====
+
+[[cleaning-up-after-a-cron-job]]
+== Cleaning Up After a Cron Job
+
+Cron jobs can leave behind artifact resources such as jobs, pods, or deployment configurations. Check if any remain:
+
+---- 
+$ oc get cronjobs
+$ oc get jobs
+$ oc get pods
+----
+
+All artifacts left over from a job execution use the job name as their prefix. For example, given the cron job example:
+
+----
+$ oc get jobs
+NAME            DESIRED   SUCCESSFUL   AGE
+pi-1497848100   1         1            1m
+pi-1497848160   1         1            49s
+
+$ oc get pods
+NAME                  READY     STATUS      RESTARTS   AGE
+pi-1497848100-lxs4k   0/1       Completed   0          2m
+pi-1497848160-6r0c8   0/1       Completed   0          59s
+----
+
+Delete each artifact if you no longer need them. To delete all jobs:
+
+----
+$ oc delete jobs --all
+----
+
+To delete all jobs created by a specific cron job:
+
+----
+$ oc delete jobs -l run=<cron_job_name>
```

Comment:
Fixed in latest commit. I added a specification for the following label in the cronjob:

>  parent="cronjobpi"

Worked fine in test.